cmd/nri-prometheus/config.go

```diff
@@ -54,6 +75,7 @@ func setViperDefaults(viper *viper.Viper) {
 	viper.SetDefault("emitter_harvest_period", "1s")
 	viper.SetDefault("auto_decorate", false)
 	viper.SetDefault("insecure_skip_verify", false)
+	viper.SetDefault("stand_alone", true)
```

Comment:
standalone should be false by default (we want POMI to behave as an on-host integration by default)
also, I would put it together "standalone", but ask Fabrizio about this.

Reply:
I'll update the default!
@theletterf What do you think about the config name? I considered it being a two words expression

Reply:
also, the standalone flag should also work as environment variable

Reply:
However I am bit worried on making as the default a new behaviour.

If they are running the integration outside K8s with a config file and they update the integration it will break since the standalone config will be not there.

Reply:
Let's go with "standalone" https://brians.wsu.edu/2016/05/31/standalone-stand-alone

Reply:
I'll add it to the dockerfile,  ENV VARS were binding already with Viper